konig-data-app-gcp/src/main/webapp/WEB-INF/cron.yaml

```diff
@@ -0,0 +1,4 @@
+cron:
```

Comment:
It is OK for this file to be hard-coded for now.  But we will eventually want this file to be automatically generated from the semantic model.  I'll add an Issue to the Issue Tracker for this functionality.